tests/unit/core/containers/TestLoginRequired.js

```diff
@@ -41,15 +42,13 @@ describe('<LoginRequired />', () => {
       store = dispatchSignInActions().store;
     });
 
-    it('sets authenticated to true when there is a token', () => {
-      expect(mapStateToProps(store.getState()))
-        .toEqual({ authenticated: true });
+    it('sets authenticated to true', () => {
+      expect(mapStateToProps(store.getState())).toEqual({ authenticated: true });
     });
 
-    it('sets authenticated to false when there is not a token', () => {
+    it('it sets authenticated to false', () => {
```

Comment:
technically the `it` isn't needed but Jest doesn't prefix our specs with "it" like mocha did so they aren't readable anyway, heh.

Reply:
oh good catch, that was actually a mistake 😞